core/misc/ui/jquery.ui.jquery-var-for-color.min.js

```diff
@@ -0,0 +1 @@
+!function(e){"use strict";"function"==typeof define&&define.amd?define(["jquery","./version"],e):e(jQuery)}(function(e){});
```

Comment:
This file doesn't seem to be referenced in the hook_libraries() list. Should it be a dependency somewhere?

Reply:
We could, but that file wouldn't do anything. See [this comment](https://github.com/backdrop/backdrop-issues/issues/5401#issuecomment-997176747) for an explanation.

Reply:
Thanks, I saw that comment but I didn't fully understand it's meaning. :+1: